.changeset/blue-sloths-stare.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': patch
+---
+
+App class now accepts renderOptions
```

Comment:
```suggestion
Adds a new `renderOptions` object  for `App/class`
```
Not sure about the syntax for App class, but it likely needs something!

Reply:
App is a class, this is for the `render` method. I'll update to something.